### DISCUSSION TITLE
Are there any mindmap size limitations for Freeplane?

### DISCUSSION THREAD

**Discussion by Coderific** [freeplane/freeplane#534] [Help] [answered]
I have some maps that are 3+ MB of text.

I opened one and copy/pasted it inside of itself and it was very slow for a log while.

In Freemind, I was able to adjust the minimum and maximum memory Java had access to to enable me to work with larger mindmaps (-Xmx4g).  Looking how to do that with Freeplane too!

Really enjoying the tool.  Just taking a bit to pick up some of the changes in keystrokes and capabilities.

**Comment by macmarrum** [comment, marked as answer]
On Linux I use `freeplane.sh` to launch the application. I can see in the script `-Xmx512m`.

My installation comes also with `freeplane.bat`, which also contains `-Xmx512m`, but if you are on Window, you probably use `freeplane.exe` instead. This is a launch4j utility, which reads `freeplane.l4j.ini`. And the ini also contains `-Xmx512m`.

This is were I would change the setting.

**Reply by Coderific** [reply]
How about for a Mac launched from the .app?  

The old freemind.app had the ability to adjust the plist. But I'm not seeing that in the current app.

Thank you for answering so quickly!  Really is different using a tool that's well supported.

**Comment by josephs07** [comment]
The `-Xmx512m` flag controls the maximum RAM usage (default is 512MB).
Change it to `-Xmx2G` to allow it to use up-to 2GB RAM. Allow more memory if needed.
